### ISSUE TITLE
Renaming the local adapter leads to an error

### ISSUE THREAD

**Issue by hytsch** [joomla/joomla-cms#40685]
### Steps to reproduce the issue
* Create an local adapter for the media manager (e.g. DP Media), e.g. "Article Document".
* Create an article and use the "Article Document" adapter. 
* Rename the local adapter, e.g. to "Article Documents".
* Open the article and try to find the previously inserted media in the media manager.

### Expected result
The local adapter is now called "Article Documents", the media are all available.

### Actual result
The local adapter is now called "Article Documents", the media have all disappeared.

### System information (as much as possible)
J 4.3.1
Content - DPMedia  1.7.1

### Additional comments
As a Joomla admin, I am used to being able to freely choose and rename names without changing the behaviour. I can rename a category, a post, a field, a module etc. without any problems. Therefore, I would autom. assume that this is also possible with a local adapter.

**Comment by laoneo** [comment]
What would help to solve this issue is a new setting which defines the title of the adapter for visual representation. Like that can the name stay in place and the title can be changed at any time. Similar to what we have with the DB related entities.

**Comment by brianteeman** [comment]
I am confused as to what this has to do with the core of Joomla. You cant rename the core joomla Filesystem-Local plugin and the DP Media filesystem plugins that I checked have a "Display Name" field

**Comment by laoneo** [comment]
Select an image in an article and then change adapter name in the local filesystem plugin.

**Comment by laoneo** [comment]
No, we were mixing up things between DPMedia and the local adapter. You were right, this is a DPMedia only issue. In the local adapter it uses automatically the selected forlder internally as name.

**Comment by laoneo** [comment]
The main issue is that the folder name is also used as label within the media manager. It would be helpful to distinguish the identifier and label. With the new label attribute it would be possible to give the adapter a name like "Sunset images", while the folder name would then be "my_beautiful_sunset_pics_from_the last_year".

**Comment by brianteeman** [comment]
To avoid confusion then I recommend that this is closed and a new feature request is opened which clearly describes what is requested

**Comment by MacJoom** [comment]
Thank you for your contribution - this Issues has been added to the issue cleanup. Please comment if you have any further input
